examples/LLM/llama2/README.md

```diff
@@ -0,0 +1,34 @@
+# Llama 2: Next generation of Meta's Language Model
+![Llama 2](./llama.png)
+
+TorchServe supports serving Llama 2 in a number of ways. The examples covered in this document range from someone new to TorchServe learning how to serve Llama 2 with an app, to an advanced user of TorchServe using micro batching and streaming response with Llama 2
+
+## 🦙💬 Llama 2 Chatbot
+
+### [Example Link](https://github.com/pytorch/serve/tree/master/examples/LLM/llama2/chat_app)
+
+This example shows how to deploy a llama2 chat app using TorchServe.
+We use [streamlit](https://github.com/streamlit/streamlit) to create the app
+
+This example is  using [llama-cpp-python](https://github.com/abetlen/llama-cpp-python).
+
+You can run this example on your laptop to understand how to use TorchServe, how to scale up/down TorchServe backend workers and play around with batch_size to see its effect on inference time
+
+![Chatbot Architecture](./chat_app/screenshots/architecture.png)
+
+## Llama 2 with HuggingFace
+
+### [Example Link](https://github.com/pytorch/serve/tree/master/examples/large_models/Huggingface_accelerate/llama2)
+
+This example shows how to serve Llama 2 - 70b model with limited resource using [HuggingFace](https://huggingface.co/meta-llama/Llama-2-70b-chat-hf). It shows the following optimizations
+    1) HuggingFace `accelerate`. This option can be activated with `low_cpu_mem_usage=True`.
+    2) Quantization from [`bitsandbytes`](https://github.com/TimDettmers/bitsandbytes)  using `load_in_8bit=True`
+The model is first created on the Meta device (with empty weights) and the state dict is then loaded inside it (shard by shard in the case of a sharded checkpoint).
+
+## Llama 2 on Inferentia
```

Comment:
Please add link for the blog which describes in more detail: https://pytorch.org/blog/high-performance-llama/

Reply:
added. Thanks! also added the picture from the blog